x/auth/client/cli/cli_test.go

```diff
@@ -145,11 +145,23 @@ func (s *IntegrationTestSuite) TestCLISign() {
 	valInfo, err := val1.ClientCtx.Keyring.Key(val1.Moniker)
 	require.NoError(err)
 
+	// query account info
+	queryResJSON, err := authtest.QueryAccountExec(val1.ClientCtx, val1.Address)
+	s.Require().NoError(err)
+	var any types.Any
+	err = val1.ClientCtx.JSONMarshaler.UnmarshalJSON(queryResJSON.Bytes(), &any)
+	var account authtypes.AccountI
+	err = val1.ClientCtx.InterfaceRegistry.UnpackAny(&any, &account)
```

Comment:
You should be able to directly unmarshal Any using `UnmarshalInterfaceJSON`:
```
var account authtypes.AccountI
err = val1.ClientCtx.JSONMarshaler.UnmarshalInterfaceJSON(queryResJSON.Bytes(), &account)
```

Reply:
oh didn't realize this exists, thanks